scratch-js/Sprite.mjs

```diff
@@ -230,6 +242,22 @@ class SpriteBase {
     }
   }
 
+  setAudioEffect(name, value) {
+    this.effectChain.setEffectValue(name, value);
+  }
+
+  changeAudioEffect(name, value) {
+    this.effectChain.changeEffectValue(name, value);
+  }
+
+  getAudioEffect(name) {
+    return this.effectChain.getEffectValue(name);
+  }
+
+  clearAudioEffects() {
+    this.effectChain.resetToInitial();
+  }
```

Comment:
Is there a way to implement something similar to the `_effectMap` so there's a more consistent API surface?

Essentially, something that can allow you to write:
```js
this.audioEffects.pitch = 50;
this.audioEffects.volume += 5;
this.audioEffects.clear()
```

as can be done with visual effects.

Reply:
Definitely agreed that making an API similar to that would be nice, both for consistency and the simpler API. I'll have a look and see if it's doable!

Reply:
Done! 547a58ea81860168ecb3e59df0b7d58f7a149b4c.